### PR TITLE
611 show tooltip method

### PR DESCRIPTION
**Pull request by GCHQ-Developer-741**
## Summary of the changes
Added new public method to change display of tooltip programmatically. Also added option to prevent tooltip from being closed by keyboard/mouse/hover events. Added new stories to react & web-components and added unit tests for this change.

## Related issue
#611 

## Checklist
- [x] I have added relevant unit and visual regression tests.
- [x] I have manually accessibility tested any changes, if relevant.
- [x] I have ensured any changes match the Figma component library.

### CODE REVIEW

**Review by GCHQ-Developer-530 — DISMISSED** [review]

File: packages/web-components/src/components/ic-tooltip/ic-tooltip.tsx [hunk 1]
```diff
@@ -39,6 +47,12 @@ export class Tooltip {
     }
   }
 
+  @Method()
+  async displayTooltip(show: boolean, persistTooltip?: boolean): Promise<void> {
+    this.persistTooltip = persistTooltip;
+    (show ? this.show : this.hide)(this.popperInstance);
```
Comment (jd239) [inline packages/web-components/src/components/ic-tooltip/ic-tooltip.tsx:53]: What does this do?

File: packages/web-components/src/components/ic-tooltip/ic-tooltip.spec.ts [hunk 2]
```diff
@@ -152,4 +152,30 @@ describe("ic-loading-indicator component", () => {
 
     page.rootInstance.disconnectedCallback();
   });
+
+  it("should open the tooltip using the displayTooltip method and should persist when persistTooltip is true", async () => {
```
Comment (jd239) [inline packages/web-components/src/components/ic-tooltip/ic-tooltip.spec.ts:156]: Create a test to testing behaviour of `displayTooltip(false, false)`;

**Comment by GCHQ-Developer-741** [comment]
> 

Have added integration tests and raised an [issue](https://github.com/mi6/ic-design-system/issues/455) to add the story to the guidance site

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Comment by MI6-451** [comment]
More a question for now. Do we think that escape should still close the tooltip when it has been opened with this method as, at least for me, it currently does not?

File: packages/web-components/src/components/ic-tooltip/ic-tooltip.tsx [hunk 3]
```diff
@@ -39,6 +47,17 @@ export class Tooltip {
     }
   }
 
+  /**
+   * Method to programmatically show/hide the tooltip without needing to physically interact with an anchor element
```
Comment (jd239) [inline packages/web-components/src/components/ic-tooltip/ic-tooltip.tsx:51]:
```suggestion
   * Method to programmatically show/hide the tooltip without needing to interact with an anchor element
```

**Review by jd239 — APPROVED** [review]

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Pull request merged**